index.html

```diff
@@ -54,8 +54,211 @@ <h2>LET'S CONNECT</h2>
+              </ul>
+            </div>
+          </div>
+    
+          <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          <div class="tech">
+            <ul>
+              <li><span>HTML</span> </li>
+              <li><span>CSS</span> </li>
+              <li><span>Javascript</span> </li>
+            </ul>
+          </div>
+        </div>
+        <button type="submit" class="btn">See Projects</button>
+      </article>
+    </section>
 
+    <section class="card-container ">
+      <article class="card card-c">
+        <div class="snap">
+          <img src="img/Snapshoot Portfolio(2).png">
+        </div>
+    
+        <div class="left-block">
+          <h5 class="tonic">Multi-Post stories </h5>
+          <div class="primary-text">
+    
+            <div class="langs">
+              <p>Canopy</p>
+              <ul>
+                <li>
+                  <div class="div"></div>
+                </li>
+                <span>Full Stack Dev</span>
+                <li>
+                  <div class="div"></div>
+                </li>
+                <span>2015</span>
+              </ul>
+            </div>
+          </div>
+    
+          <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          <div class="tech">
+            <ul>
+              <li><span>HTML</span> </li>
+              <li><span>CSS</span> </li>
+              <li><span>Javascript</span> </li>
+            </ul>
+          </div>
+        </div>
+        <button type="submit" class="btn">See Projects</button>
+      </article>
+    </section>
 
+    <section class="card-container ">
+      <article class="card card-c">
+        <div class="snap">
+          <img src="img/Snapshoot Portfolio2.png">
+        </div>
+    
+        <div class="left-block">
+        <h5 class="tonic">Multi-Post stories </h5>
+          <div class="primary-text">
+    
+            <div class="langs">
+              <p>Canopy</p>
+              <ul>
+                <li>
+                  <div class="div"></div>
+                </li>
+                <span>Full Stack Dev</span>
+                <li>
+                  <div class="div"></div>
+                </li>
+                <span>2015</span>
+              </ul>
+            </div>
+          </div>
+    
+          <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          <div class="tech">
+            <ul>
+              <li><span>HTML</span> </li>
+              <li><span>CSS</span> </li>
+              <li><span>Javascript</span> </li>
+            </ul>
+          </div>
+        </div>
+        <button type="submit" class="btn">See Projects</button>
+      </article>
+    </section>
 
+  
+    <section class="container-b">
+      <article class="card-b">
+        <h1 class="about-h1">About Myself</h1>
```

Comment:
- [ ] As mentioned previously kindly make sure to only use one `h1` element per page in accordance with the proper HTML semantics. ✔
- [ ] As I mentioned previously make sure to import the proper font weights and adjust your design so it looks more like the Figma design.
- Your design:
  ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/104968236/78cb59d8-8298-4387-a8ed-4247a9803aba)
- Figma design:
  ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/104968236/5b4ff520-a4e2-4c05-87e8-330ed81a1328)
- [ ] Once again as mentioned previously there seems to be a discrepancy between your design and the Figma design regarding the spacing of the elements as shown below so please work on them.
- Your design:  
  ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/104968236/7a8590f7-c907-4021-874a-69de481429a2)
- Figma design:
  ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/104968236/257e08ea-050c-431c-9353-00c4694cf23c)